resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,70 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*Id" : "r",
+        ".*\\.id" : "r",
+        ".*\\.data" : "r",
         ".*\\.description" : "cru",
         ".*\\.isPublic" : "cru",
-        ".*\\.kind" : "cr",
+        ".*\\.kind" : "r",
         ".*\\.name" : "cru",
         ".*\\.removeTime" : "ru",
-        ".*\\.uuid" : "cr",
-
+        ".*\\.uuid" : "r",
+        ".*\\.accountId" : "r",
+        ".*\\.created" : "r",
+        ".*\\.state" : "r",
+        ".*\\.removed" : "r",
+        ".*\\.transitioning" : "r",
+        ".*\\.transitioningMessage" : "r",
+        ".*\\.transitioningProgress" : "r",
+        
+        "account" : "crud",
+        "account.id" : "cru",
         "account.externalId" : "cru",
         "account.externalIdType" : "cru",
-        "account.defaultCredentialIds" : "cru",
-        "account.defaultNetworkIds" : "cru",
+        "account.kind" : "cr",
+        "account.uuid" : "cr",
 
         "activeSetting" : "rud",
         "activeSetting.name" : "cr",
         "activeSetting.value" : "ru",
 
-        "agent.managedConfig" : "cru",
-        "agent.uri" : "cru",
-        "agent.user" : "cr",
-
-        "agentInstanceProvider.agentInstanceImageUuid" : "cru",
-
-        "configItemStatus.appliedUpdated" : "cru",
-        "configItemStatus.appliedVersion" : "cru",
-        "configItemStatus.name" : "cru",
-        "configItemStatus.requestedUpdated" : "cru",
-        "configItemStatus.requestedVersion" : "cru",
-        "configItemStatus.sourceVersion" : "cru",
-
-        "configItem.sourceVersion" : "cru",
-
+        "agent" : "r",
+        "agent.accountId" : "r",
+        "agent.managedConfig" : "r",
+        "agent.uri" : "r",
+        "agent.agentGroupId": "",
+        "agent.zoneId" : "",
+        "agent.user" : "",
+        "agent.removeTime" : "",
+        "agent.data" : "",
+            
+        "apiKey" : "crud", 
+        "apiKey.accountId" : "cr",
```

Comment:
Calling out that this looks suspicious and doesnt match the spreadsheet. @pranavs18 and @ibuildthecloud discuss and C is needed for tests to pass.